crypto/fipsmodule/digest/internal.h

```diff
@@ -85,6 +85,10 @@ struct env_md_st {
   // final completes the hash and writes |md_size| bytes of digest to |out|.
   void (*final)(EVP_MD_CTX *ctx, uint8_t *out);
 
+  // finalXOF completes the hash and writes |len| bytes of digest extended output
+  // to |out|.
+  void (*finalXOF)(EVP_MD_CTX *ctx, uint8_t *out, size_t len);
```

Comment:
Looks like our formal verification isn't happy about this struct field being added:
<img width="1013" alt="Screenshot 2023-09-19 at 12 19 07 PM" src="https://github.com/aws/aws-lc/assets/3589880/63fac3d9-b1f3-4342-9120-77ebc7ce805b">

Reply:
Moving this field to the end of the struct appears to have resolved the backwards incompatibility.